src/main/java/seedu/address/commons/events/ui/DeselectRoomListEvent.java

```diff
@@ -0,0 +1,15 @@
+package seedu.address.commons.events.ui;
+
+import seedu.address.commons.events.BaseEvent;
+
+/**
+ * Represents a deselect room list event
+ */
+public class DeselectRoomListEvent extends BaseEvent {
```

Comment:
In what cases would this event be used? A room is never actually "deleted" from Concierge after all.

Reply:
yup you're right. I forgot to remove it after coming to the same conclusion just now.

Reply:
fixed in latest commit